src/front/app/components/Challenges/index.js

```diff
@@ -3,10 +3,34 @@ import ChallengeList from './ChallengeList';
 import Menu from './Menu';
 import Navigation from './../Navigation';
 
-export default () => (
-    <div>
-        <Menu />
-        <ChallengeList />
-        <Navigation />
-    </div>
-);
+class Challenges extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            menuClass: 'challenges__menu'
+        };
+    }
+
+    onScroll(e) {
+        e.preventDefault();
+        if (e.target.scrollTop) {
+            this.setState({ menuClass: 'challenges__menu_active' });
+        } else {
+            this.setState({ menuClass: 'challenges__menu' });
+        }
+    }
+
+    render() {
+        return (
+            /* eslint-disable react/jsx-no-bind  */
```

Comment:
Disabling of eslint rules should be a rare exception - in this case binding (always) should be done in the constructor function, as it's suggested in the rule - please read our styleguide https://github.com/airbnb/javascript/tree/master/react#naming to find the reasons behind the rule